libzecale/circuits/verifier_gagdet_imports.hpp

```diff
@@ -24,7 +24,7 @@ using VerificationKeyVariableGadgetT =
 
 #ifdef ZKSNARK_GROTH16
 #define LIBZECALE_VERIFIER_GADGET_DEFINED
-#include "libzecale/circuits/groth16_verifier/r1cs_gg_ppzksnark_verifier_gagdet.hpp"
+#include "libzecale/circuits/groth16_verifier/r1cs_gg_ppzksnark_verifier_gadget.hpp"
```

Comment:
Hum.. good catch for this typo.. I guess/hope the groth verifier gadget was imported somewhere else, otherwise I wonder how the project compiled :thinking:

Reply:
It compiled fine, because the file was named like this.

Reply:
Oh was it?... Ok that's "reassuring" then. Thanks for fixing the typo